### DISCUSSION TITLE
Issue with GitHub Student Pack Application

### DISCUSSION THREAD

**Discussion by laithalhelwane** [community/community#136863] [GitHub Education] [answered]
### Hi everyone,

I’m having some trouble applying for the GitHub Student Developer Pack and I’m hoping someone might be able to help. I've made sure that I meet all the requirements and submitted valid proof of my student status, but for some reason, my application isn't going through. Unfortunately, I haven't received any specific information on what might be causing the issue.

Has anyone else experienced this, or does anyone have any advice on what I could do to resolve it? Any guidance would be much appreciated!

Thanks in advance!
![image](https://github.com/user-attachments/assets/009974ba-2a20-4dcc-9b0c-8da5c7c86d26)

**Comment by davevad93** [comment, marked as answer]
Hi @laithalhelwane ,

I've never experienced that and considering that there are no details about the issue with your profile, I suggest you to contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other`, attach this same screenshot and explain the problem with your application.

**Reply by laithalhelwane** [reply]
Thanks
